modules/aws-frontend/variables.tf

```diff
@@ -56,3 +56,9 @@ variable "content_security_policy" {
     "worker-src"  = "blob:"
   }
 }
+
+variable "web_acl_id" {
+  type        = string
+  default     = ""
```

Comment:
So what do we do about it?

Reply:
I forgot to include the updated file in the commit, sorry :)